google_takeout_parser/parse_json.py

```diff
@@ -156,8 +156,8 @@ def _parse_location_history(p: Path) -> Iterator[Res[Location]]:
         source = loc.get("source")
         try:
             yield Location(
-                lng=float(loc["longitudeE7"]) / 1e7,
-                lat=float(loc["latitudeE7"]) / 1e7,
+                lng=float(loc.get("longitudeE7")) / 1e7,
```

Comment:
yeah, so these should probably just be changed back. `float(None)` is just an error, and its more confusing than there being a `KeyError` because google changed the keys.